catalogue/webapp/components/Work/Work.tsx

```diff
@@ -38,7 +39,7 @@ type Props = {
 };
 
 const Work = ({ work }: Props) => {
-  const { archivesPrototype } = useContext(TogglesContext);
+  const { archivesPrototype, archivesPrototypeSidePanel } = useContext(TogglesContext);
```

Comment:
Is there a reason to have both here? We seem to get into toggle-spaghetti quite often, why not just have them under 1?

Reply:
I _think_ `archivesPrototype` will load the tree in a modal window, whereas `archivesPrototypeSidePanel` loads it on the side of the page by default. Not sure if there are other reasons (do they rely on anything different from an API perspective), but I believe we'd want to be able to have both. I'll take a look, though.

Reply:
Ah, I see - thanks for explaining. It would be good in future to approach each toggle as separate e.g.
```tsx
{archiveModal && <ArchiveModal />}
{archiveSidepanel && <ArchiveSidepanel />}
```

Toggles should really touch as little live code as possible otherwise they're not really toggles.